package.json

```diff
@@ -27,24 +27,24 @@
         "release": "release-it"
     },
     "dependencies": {
-        "eslint-utils": "^3.0.0",
-        "rambda": "^7.4.0"
+        "eslint-utils": "3.0.0",
+        "rambda": "7.4.0"
```

Comment:
Please keep the ranges for regular dependencies. So dependency deduplication works for consumers.

Reply:
Done ✅